### PR TITLE
scaffold cart

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: templates/cart.liquid [hunk 1]
```diff
@@ -1,33 +1,50 @@
...
+<section id="cart">
+  <div class="container">
+    {% if cart.item_count > 0 %}
+    <form action="/cart" method="post" id="cart" class="cart">
+
+      <div class="grid-row">
+        {% for item in cart.items %}
+        <div id="cart-item-{{item.variant.id}}" class="cart-item grid-item item-s-12 no-gutter grid-row align-items-end margin-bottom-small">
+          <div class="grid-item item-s-2 item-m-3 offset-m-0 item-l-2 offset-l-1">
+            <a href="{{ item.url }}">
```
Comment (varffs) [inline templates/cart.liquid]: Not sure why but this link doesn't seem to work

**Review by varffs — CHANGES_REQUESTED** [review]
I also think the scaffold should include the responsive layout.

**Comment by mikeyrayvon** [comment]
shop-apt-25 myshopify com-cart iphone 5

it includes a responsive layout

**Comment by mikeyrayvon** [comment]
omg enormous pic

**Comment by mikeyrayvon** [comment]
i guess thats not a great responsive layout. if you want to try something else i'm ok

**Review by varffs — APPROVED** [review]

**Comment by varffs** [comment]
We can leave the responsive details for later. But I think we should generally do something in the scaffold when we can.

**Pull request merged**